app/javascript/schools/settings/components/SchoolCustomize__LinksEditor.res

```diff
@@ -1,3 +1,4 @@
+open SchoolCustomize__Types.SchoolLinks
```

Comment:
This module need not be opened as we already have it in `SchoolCustomize__Types`

Reply:
@mahesh-krishnakumar Rescript throws an error if  `open SchoolCustomize__Types.SchoolLinks` is removed